_config/subsite.yml

```diff
@@ -6,6 +6,6 @@ Only:
 UserSecurityReport:
   extensions:
```

Comment:
I wonder if there's a way to ignore this yaml... if nothing else, comment it all out?